.github/workflows/job-dispatch.yml

```
@@ -0,0 +1,84 @@
+        uses: peter-evans/repository-dispatch@v3
+        with:
+          token: ${{ secrets.KUBERAY_DISPATCH_TESTS }}
+          repository: project-codeflare/kuberay-post-merge-tests
+          event-type: branch-dispatch
+
+      - name: Poll Project-Codeflare dummy repo for workflow completion
+        env:
+          GH_TOKEN: ${{ secrets.PAT_KEY }}
+          ORG: project-codeflare
+          REPO: kuberay-post-merge-tests
+          BRANCH: ${{ inputs.branch }}
```

actionlint (1.7.7)

property "branch" is not defined in object type {}

(expression)